src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,205 @@
+                "Test failed! Expected " + WrongDataException.class.getName()
+                        + " Login cannot be empty!");
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        User user = new User(123456L, null, "qwerty", 21);
+        assertThrows(WrongDataException.class, () -> registrationService.register(user),
+                "Test failed! Expected " + WrongDataException.class.getName()
+                        + " Login cannot be null!");
+    }
+
+    @Test
+    void register_toLongLogin_notOk() {
+        User user = new User(123456L, "Polischuck", "qwerty", 21);
+        assertThrows(WrongDataException.class, () -> registrationService.register(user),
+                "Test failed! Expected " + WrongDataException.class.getName()
+                        + " Login must contain no more " + 6 + " characters");
+    }
+
+    @Test
+    void register_loginWithSpace_notOk() {
+        User user = new User(123456L, " Polis", "qwerty", 21);
+        assertThrows(WrongDataException.class, () -> registrationService.register(user),
+                "Test failed! Expected " + WrongDataException.class.getName()
+                        + " Login cannot contain spaces!");
+    }
+
+    @Test
+    void register_successRegister_ok() {
+        User user = new User(123456L, "Ford", "qwerty", 21);
+        User actual = registrationService.register(user);
+        assertEquals(user, actual, "Registration must be successful!");
+    }
+
+    @Test
+    void get_user_ok() {
+        User user = new User(123456L, "Second", "qwerty", 21);
+        registrationService.register(user);
+        User actual = storageDao.get("Second");
+        assertEquals(user, actual, "The method must return " + user);
+    }
+
+    @Test
+    void storage_countOfUser_ok() {
+        User user = new User(123456L, "Red", "qwerty", 21);
+        User user1 = new User(123457L, "Blue", "qwe123", 27);
+        User user2 = new User(123458L, "Yellow", "123rty", 29);
+        registrationService.register(user);
+        registrationService.register(user1);
+        registrationService.register(user2);
+        int actual = Storage.people.size();
+        assertEquals(actual, 3,
+                "Incorrect number of users! Expected " + 3 + ", but was " + actual);
+    }
+
+    @Test
+    void equals_usersEqual_ok() {
+        User user = new User(223456L, "Green", "qwerty", 24);
+        User user1 = new User(223456L, "Green", "qwerty", 24);
+        boolean actual = user.equals(user1);
+        assertTrue(actual,
+                "Users must be equal! Expected " + true + ", but was " + actual);
+    }
+
+    @Test
+    void equals_usersNotEqual_NotOk() {
+        User user = new User(223456L, "Green", "qwerty", 24);
+        User user1 = new User(193456L, "Yellow", "qwerty", 24);
+        boolean actual = user.equals(user1);
+        assertFalse(actual,
+                "Users must be not equal! Expected " + false + ", but was " + actual);
+    }
```

Comment:
there is no point in testing equals()
also this class  should test only methods of RegistrationServiceImpl (register() method)
so let's remove this